toolchain/lower/testdata/function/call/struct_param.carbon

```diff
@@ -13,7 +13,7 @@ fn Main() {
 // CHECK:STDOUT: ; ModuleID = 'struct_param.carbon'
 // CHECK:STDOUT: source_filename = "struct_param.carbon"
 // CHECK:STDOUT:
-// CHECK:STDOUT: @struct = internal constant { i32, i32 } { i32 2, i32 3 }
```

Comment:
It's curious to me that the `@struct`, `@tuple` names are lost. Is that something that can be avoided? It feels like it reduces clarity of the LLVM IR.

Reply:
We can make `InstNamer` include them in the names of constants. I'll send a separate PR for that.